Demos/setup.py

```diff
@@ -8,18 +8,31 @@
 from distutils.extension import Extension
 from Cython.Build import cythonize
 
-ext_modules = cythonize("**/*.pyx", exclude="numpy_*.pyx")
+ext_modules = cythonize("**/*.pyx",
+                            exclude=["numpy_*.pyx",
+                                     "callback/*.pyx",
+                                     "libraries/*.pyx"])
 
 # Only compile the following if numpy is installed.
 try:
     from numpy import get_include
 except ImportError:
     pass
 else:
-    numpy_demo = [Extension("*",
+    ext_def = [Extension("cheese",
+                            sources=["callback/*.pyx",
+                                     "callback/cheesefinder.c"],
+                            include_dirs=[get_include(),
+                                          "callback"]),
```

Comment:
The "cheese" extension doesn't involve numpy so doesn't need to be in the `else` clause of `try: from numpy import get_include`